docs/components/Address.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 1
+---
+
+# Address
+
+Display an address (or ENS) plus a copy address icon utility. If the address has an ENS with avatar associated, it will show this avatar, if not it will show a blockie image for that address.
+
+By default, clicking on the address sends you to blockexplorer details for that address.
+
+![Address Example](/img/Address.png)
+
+## Import
+
+```ts
+import { Address } from "~~/components/scaffold-eth";
+```
+
+## Usage
+
+```ts
+const [selectedAddress, setSelectedAddress] = useState("");
+
+<Address address={selectedAddress} />;
+```
+
+## Props
+
+- `address` => Address in `0x___` format, it will resolve its ENS if has one associated.
+
+- `disableAddressLink` => Set it to `true` to disable the blockexplorer link behaviour when clicking on the address.
+
+- `format` => By default, displayed address it's only the first five characters from the address, set this to `"long"` to display the full address.
```

Comment:
We can also add `size` prop : 

`size` (optional) => default to `base` but you can pass in `xs`, `sm`, `base`, `lg`, `xl`, `2xl`, `3xl`

Reply:
Totally skipped this prop! Love the (optional) after prop name. Just pushed the changes on this page, setting:

`size` (optional) => Size for the displayed Address component. `base` by default but you can pass in `xs`, `sm`, `base`, `lg`, `xl`, `2xl`, `3xl`.